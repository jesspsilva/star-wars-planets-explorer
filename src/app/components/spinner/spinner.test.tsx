import { render } from "@testing-library/react";

import Spinner from "./spinner";

describe("Spinner component", () => {
  it("matches snapshot", () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
