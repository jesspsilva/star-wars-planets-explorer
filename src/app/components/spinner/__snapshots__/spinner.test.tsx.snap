// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component matches snapshot 1`] = `
<div>
  <div
    class="sc-gtLWhw fTXrlL"
    data-testid="loading-spinner"
  >
    <div
      class="sc-blHHSb kvKaeK"
    />
  </div>
</div>
`;
